modules/ROOT/pages/access-client-app-id-task.adoc

```diff
@@ -3,7 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-As an owner of a xref:exchange::to-request-access.adoc[registered client application], you are sometimes required to provide the client ID and the client secret of the application when accessing an API. After a contract is created between the client application and the API, you can access the API using these credentials based on the policies that are applied. For example, if the API owner has applied the Client ID Enforcement policy on the API, you must provide the client ID, and optionally, the client secret information for the client application. 
+As the owner of a xref:exchange::to-request-access.adoc[registered client application], you are occasionally required to provide your client ID and client secret (optional) when accessing an API, after a contract is created between your application and an API. 
```

Comment:
This paragraph is a bit confusing. If you are creating a Client Application, is probably because the contract is required to consume the API. It is not an optional parameter.

Reply:
Done! Thanks!

Reply:
Done! Thanks!